OracleDataIntegrator/README.md

```diff
@@ -37,7 +37,7 @@ Sample Data will look like this...
         ORACLE_PWD=Welcome1
         
 The Database 12.2.0.1 container can be started from the [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase/dockerfiles/12.2.0.1). 
```

Comment:
This could be rewritten as, "You can build an [Oracle Database 12c Enterprise Edition image](https://github.com/oracle/docker-images/tree/master/OracleDatabase/dockerfiles/12.2.0.1) or download a pre-built image from the Oracle Container Registry or the Docker Store.